### ISSUE TITLE
Can't use keyboard arrow to move around text inputs

### ISSUE THREAD

**Issue by zplawecki** [zen-browser/desktop#3834]
### Captchas

- [X] I have read the instructions.
- [X] I have searched existing issues and avoided creating duplicates.
- [X] I am not filing an enhancement request.

### What happened?

Anytime I want to write a text in some kind of text input (facebook, github, mastodon, any other), I can't use keyboard or shortcuts like shift+end to navigate on text 

### Reproducible?

- [X] I have checked that this issue cannot be reproduced on Mozilla Firefox.

### Version

1.0.2-b.2

### What platform are you seeing the problem on?

Windows

**Comment by mr-cheffy** [comment]
Is this a feature request?

**Comment by pixincreate** [comment]
<strike>
this is not a feature request. i can reproduce this on mac.
basically, tab button, arrow keys, just do not work.
</strike>


edit: restarting the app fixes this

**Comment by jonaskohl** [comment]
Same issue here. Only happens sometimes and it fixes itself when quitting and re-opening Zen. I have a feeling that this mainly occurs after the computer went to sleep. This has only been an issue recently (only since 1.0.2-b0 or b1, I can't really tell). I'm on macOS 14.7.1

**Comment by zplawecki** [comment]
I closed this one because I thought it fixed itself, but it must have been just case after restart, bu it keeps coming back

**Comment by d0tlake** [comment]
Also having this issue, and it also seems to affect different kinds of shortcuts such as ctrl+tab or shift+ctrl+tab for tab navigation and I also can't use the Home or End keys for page or text navigation. Doesn't affect all shortcuts or keys though, since I can do things like open split view or use PgUp or PgDn.

One interesting thing to note, is that it seems to only happen in one window. I currently have two windows up, and I am only getting the issue in one of the windows, but not the other. Restarting the browser fixes the issue, and not sure what causes the problem as it seems to randomly happen eventually.

**Comment by BoscoDomingo** [comment]
Same. Can't use Tab to jump across fields either. I just upgraded to the beta from Alpha 17 and I'm regretting it just for this, as I'm constantly having to restart just for the keyboard to work for a bit. Writing this very message was harder than it needed to as I had to move around with the mouse instead of keyboard

**Comment by gvijayakumar-16** [comment]
Facing the same issue. Started happening only after the update to beta version.
Cycling through tabs, arrow keys, refresh are a few of the functionalities that are not working via keyboard keys.
Restarting the app fixes the issue though

**Comment by jonaskohl** [comment]
Also, I noticed opening DevTools triggers this bug as well

**Comment by ClixTW** [comment]
I have discovered a 100% reproducible method. Click the extension icon, select any extension, and then the keyboard shortcuts will become unusable.

Edit: Not all shortcuts are affected, but at least Ctrl+Tab for switching tabs and the arrow keys for moving the cursor become unusable.

**Comment by kuqquk** [comment]
Confirmed in 1.0.2-b.3 (64-bit) Windows version...... So painful!

> I have discovered a 100% reproducible method. Click the extension icon, select any extension, and then the keyboard shortcuts will become unusable.
> 
> Edit: Not all shortcuts are affected, but at least Ctrl+Tab for switching tabs and the arrow keys for moving the cursor become unusable.

**Comment by franksodre** [comment]
Is the same for me on windows. i can't use the arrow keys, the **Home**, **End** even **tab** too

**Comment by rafa-ulloa** [comment]
same issue on Pop!_OS. arrows and tab key unresponsive

**Comment by Outshine8** [comment]
The same issue has been happening for a while going on some streaming sites, sometimes it'll just happen by itself too.

Also, the Numpad arrow keys work

**Comment by EgoisticEscapist** [comment]
Same issue. Can't use arrow keys on Windows 11 after clicking any extension in 1.0.2-b.3 :(

**Comment by mr-cheffy** [comment]
Duplicate of https://github.com/zen-browser/desktop/issues/2743
